base/deprecated.jl

```diff
@@ -1632,6 +1632,12 @@ function SymTridiagonal(dv::AbstractVector{T}, ev::AbstractVector{S}) where {T,S
     SymTridiagonal(convert(Vector{R}, dv), convert(Vector{R}, ev))
 end
 
+# issue #22791
+@deprecate_binding select partialsort
```

Comment:
You get different stack traces, and to some extent different information:
```julia
julia> foo() = 1;

julia> Base.@deprecate_binding bar foo

julia> bar()
WARNING: Main.bar is deprecated, use Main.foo instead.
  likely near no file:0
1
```

vs

```julia
julia> foo() = 1;

julia> Base.@deprecate bar foo
:bar

julia> bar()
WARNING: bar is deprecated, use foo instead.
Stacktrace:
 [1] depwarn at ./deprecated.jl:70 [inlined]
 [2] bar() at ./deprecated.jl:31
 [3] eval(::Module, ::Expr) at ./repl/REPL.jl:3
 [4] eval_user_input(::Any, ::Base.REPL.REPLBackend) at ./repl/REPL.jl:69
 [5] macro expansion at ./repl/REPL.jl:100 [inlined]
 [6] (::Base.REPL.##1#2{Base.REPL.REPLBackend})() at ./event.jl:73
while loading no file, in expression starting on line 0
1
```